### DISCUSSION TITLE
VegX to TERN NDES - modelling cookbook use case

### DISCUSSION THREAD

**Discussion by gwittich** [ternaustralia/ontology_tern#51] [Modelling]
Please include a mapping of:
VegX to TERN NDES
in the cookbook as a priority.

Gail on Behalf of Gaia Resources.

**Comment by smguru** [comment]
@gwittich can you please provide data in VegX format so that we can work on the crosswalk.

**Comment by gwittich** [comment]
Hi Guru, I understand we are yet to supply this data. We can leave this on the back burner until we have resolved the DwC points. Is that suitable to you?

**Reply by smguru** [reply]
thanks Gail, yes that should be fine.
